keps/sig-network/2030-topology-aware-proxying/README.md

```diff
@@ -245,6 +230,134 @@ not reliant on any other feature being enabled in any other release. [KEP
 #2004](https://github.com/kubernetes/enhancements/issues/2004) will be dependent
 on this KEP though.
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: TopologyAwareProxying
+    - Components depending on the feature gate: kube-proxy
+
+* **Does enabling the feature change any default behavior?**
+  Not really. Kube-Proxy will still consume endpoints without these topology
+  labels (the default). When these labels are set, kube-proxy will only consume
+  a subset of endpoints closest to it. This will not be noticeable in most
+  cases.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Yes - kube-proxy will simply revert to watching all EndpointSlices.
```

Comment:
Please update - we decided that you ill be watching everything in all cases.

Reply:
Updated, thanks!